actfw_core/unicam_isp_capture.py

```diff
@@ -31,8 +38,34 @@
 # TODO: support other than imx219
 # pick from https://github.com/kbingham/libcamera/blob/22ffeae04de2e7ce6b2476a35233c790beafb67f/src/ipa/raspberrypi/data/imx219.json#L132-L142 # noqa: E501, B950
 
-V1_UNICAM_SIZES: List[Tuple[int, int]] = [(2592, 1944), (1920, 1080), (1296, 972), (640, 480)]
-V2_UNICAM_SIZES: List[Tuple[int, int]] = [(3280, 2464), (1920, 1080), (1640, 1232), (640, 480)]
+MAX_LS_GRID_SIZE = 0x8000
+LS_TABLE_W = 16
+LS_TABLE_H = 12
+
+
+# correspond to [libcamera's CameraMode](https://github.com/raspberrypi/libcamera/blob/3fad116f89e0d3497567043cbf6d8c49f1c102db/src/ipa/raspberrypi/controller/camera_mode.h#L19) # noqa: E501, B950
+@dataclass(frozen=True, eq=True)
```

Comment:
Python 3.6 does not support dataclass. Something wrong may happen in jetson.

https://docs.python.org/ja/3.10/library/dataclasses.html

Reply:
actfw-core depends `dataclasses` package in python 3.6 environment, so it seems ok.
https://github.com/Idein/actfw-core/blob/master/pyproject.toml#L38

also dataclass is already used [here](https://github.com/Idein/actfw-core/blob/3587d964011007a1e153ac77941a81161f8403ea/actfw_core/_private/schema/agent_app_protocol.py#L26)